app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+class MoviesController < ApplicationController
+  include JSONAPI::Fetching
+  skip_before_action :authenticate_user!, only: %i[show index]
+
+  def index
+    movies = Movie.all
+
+    render jsonapi: movies, except: blacklisted_attributes
```

Comment:
I think it would be more suistnable to create a Service for representing data. WDYT?

Reply:
I used similar logic with blacklisted attributes in the other controller so I wanted to be consistent.

Reply:
Fair enough :)